.github/mergify.yml

```
@@ -96,7 +96,8 @@ pull_request_rules:
           - invalid
       queue:
         name: edge-api
-        method: merge
+        method: squash
+        commit_message_template: "{{ commits[0] }}"
```

```suggestion
        commit_message_template: {{ commits[0] }}
```
I think should be without quotes

---

it does not work wihout quotes